src/main/java/org/broadinstitute/hellbender/tools/copynumber/gcnv/GenerateVCFFromPosteriors.java

```diff
@@ -0,0 +1,224 @@
+package org.broadinstitute.hellbender.tools.copynumber.gcnv;
+
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.variant.variantcontext.writer.VariantContextWriter;
+import org.apache.commons.lang3.tuple.ImmutableTriple;
+import org.apache.commons.lang3.tuple.Triple;
+import org.apache.log4j.LogManager;
```

Comment:
Is this the correct logger?  We use `org.apache.logging.log4j.LogManager` elsewhere.  @jonn-smith I noticed that Funcotator and XsvLocatableTableCodec also use this logger.

Reply:
Done